appveyor.yml

```diff
@@ -0,0 +1,14 @@
+environment:
+    DEPLOY_PROVIDER: bintray
+    BINTRAY_ACCOUNT: alexpux
+    BINTRAY_REPOSITORY: msys2
+    PACMAN_REPOSITORY_NAME: ci.msys
+
+build_script:
+    - C:\msys64\usr\bin\pacman --noconfirm --sync --refresh --refresh --sysupgrade --sysupgrade
+
+test_script:
+    - git submodule init
```

Comment:
This needs to be `git submodule update --init` to actually checkout the submodule. Looks like this is why your appveyor test is failing.